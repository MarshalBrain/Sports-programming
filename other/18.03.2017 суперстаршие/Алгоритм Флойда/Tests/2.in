10
0 46 40 24 95 95 41 87 5 3 
72 0 14 38 83 60 93 100 17 18 
54 48 0 0 62 20 8 21 36 36 
59 68 87 0 29 89 92 79 78 40 
50 82 29 61 0 68 1 58 17 30 
69 99 70 52 57 0 70 37 100 38 
19 78 69 36 12 90 0 96 62 86 
8 84 56 89 39 75 41 0 84 9 
52 47 73 66 62 93 86 72 0 89 
65 6 72 62 8 29 4 46 92 0 
