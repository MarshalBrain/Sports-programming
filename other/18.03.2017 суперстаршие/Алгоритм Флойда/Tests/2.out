0 9 23 24 11 32 7 44 5 3 
68 0 14 38 26 34 22 35 17 18 
27 36 0 44 20 20 8 21 32 30 
59 46 58 0 29 69 30 79 46 40 
20 29 29 37 0 49 1 50 17 23 
45 44 58 52 46 0 42 37 50 38 
19 28 41 36 12 51 0 62 24 22 
8 15 29 32 17 38 13 0 13 9 
52 47 61 66 62 81 59 72 0 55 
23 6 20 40 8 29 4 41 23 0 
